packages/@react-spectrum/table/test/TableDnd.test.js

```diff
@@ -3053,24 +3077,22 @@ describe('TableView', function () {
       expect(rows).toHaveLength(6);
 
       // Select the folder and perform a drag. Drag start shouldn't include the previously selected items
-      moveFocus('ArrowDown');
-      fireEvent.keyDown(document.activeElement, {key: 'Enter'});
-      fireEvent.keyUp(document.activeElement, {key: 'Enter'});
+      await user.keyboard('{ArrowDown}');
+      await user.keyboard('{Enter}');
       // Selection change event still has all keys
       expect(new Set(onSelectionChange.mock.calls[0][0])).toEqual(new Set(['1', '2', '3', '8']));
 
       draghandle = within(rows[0]).getAllByRole('button')[0];
       expect(draghandle).toBeTruthy();
       expect(draghandle).toHaveAttribute('draggable', 'true');
-      moveFocus('ArrowRight');
-      fireEvent.keyDown(draghandle, {key: 'Enter'});
-      fireEvent.keyUp(draghandle, {key: 'Enter'});
+      await user.keyboard('{ArrowRight}');
+      await user.keyboard('{Enter}');
```

Comment:
Nit: the intent of this portion of the test was to make sure that triggering drag on a non-selected item didn't include selected rows (i.e. it selects the row with `key='8'` then start a drag via row with `key='0'` and ensures that drag start only includes `key='0'`). As it is now, this drags the same row that is selected which is already tested above. I'll make the comment above clearer and see about making this change after finishing my sweep through the other tests